molecule/amqp_connection/molecule.yml

```diff
@@ -4,7 +4,7 @@ dependency:
   options:
     role-file: molecule/common/requirements.yml
 driver:
-  name: podman
+  name: docker
```

Comment:
:(

Reply:
The molecule github action only supports docker.
The alternative would be to use the run module with a bunch of pre-install commands to install and configure both docker and podman, and that led to a bunch of complaints from the GHA system about conflicts and so on.

Reply:
Oh yea I know there isn't really anything you can do about it, it's sad nonetheless :D